### ISSUE TITLE
Multi-GPU question

### ISSUE THREAD

**Issue by kireita** [joeknock90/Single-GPU-Passthrough#63]
hey,

i have the following case:
2 gpu's a 1050 and a 1060, both on the low end, i have my displays connected to the 1050 and now i just pass through my 1060 to the vm, but i never have control over my 1060. is it possible to adjust the script to just enable vfio passthrough for my 1060 without restarting x? what i want is: to enable vfio when i start my vm and disable it when i stop using it but i dont want to restart my workspaces everytime i start or stop my vm.

TL:DR i want my 1060 always avaible to me while using my 1050 as the main screen provider when i turn on my vm i pass the GPU and when i turn it off its avaiable again in linux

would this be possible? if so.Can you point me to some direction for read about it.

**Comment by Ohio2** [comment]
That's a job for the KVM switch.
